config/module.config.php

```diff
@@ -50,6 +50,19 @@
     'zf-content-validation' => [
         'methods_without_bodies' => [],
         /*
+         * Use this config to validate parameters from routes
+         *
+         * Only configured routes and parameters will validated
+         *
+         * Format follows:
+         *
+         * '<route name>' => [
+         *  '<route.param.1>',
+         *  '<route.param.2>'
+         * ]
```

Comment:
Would this be an associative or an indexed array?

As an example, given the route named "user_contact" with the string `/api/user/:user_id/contact/:contact_id`, would this be:

```php
'user_contact' => [
    "/some-regex-for-user_id",
    "/some-regex-for-contact_id",
]
```

or

```php
'user_contact' => [
    'user_id' => 'regexp',
    'contact_id' => 'regexp',
]
```

Second question: are the values intended to be regexp, or named validator services, or the name of an input? Based on the tests, my understanding is that it's the latter, but the description above is not clear.